src/managers/TenorManager.ts

```diff
@@ -61,7 +60,7 @@ class TenorManager {
 	}
 
 	private praseResult(img: any): TenorImage {
-		const gif = img['media_formats']['gif'];
+		const gif = img['media_formats']['tinygif'];
```

Comment:
tinygif can be used for preview and you are returning it to the end user.  You should return the full gif to callback and use these tiny ones for previews in picker.

Reply:
No, this is what's causing the browser to load the full gifs since this method is called on search and trending. The gif object returned as all the needed information to get the full resolution gif. The URL is the full resolution and can be used by clients img src or however needed.

Reply:
this is the only place where `TenorImage` objects are generated and then passed to `onGifClick`

Reply:
current version clicking trending: ![image](https://user-images.githubusercontent.com/107433287/211908049-b3ad77d4-090e-4616-b0b4-d749f8f65b2c.png)

using tinygif:
![image](https://user-images.githubusercontent.com/107433287/211908286-663f6b19-a311-4ba7-8174-6d82908fb4c3.png)

Reply:
![image](https://user-images.githubusercontent.com/23432278/211909305-5228c9f7-00ed-4ed1-ad3a-7773637f71e9.png)

![image](https://user-images.githubusercontent.com/23432278/211909339-2d6d84ea-fbe8-4b9e-8399-221dc919e790.png)

Reply:
Updated to use fullgif dimensions. You can see that when specifying to use the tiny gif the netwrok request size in bytes is drastically reduced. Using the full gif size will give the callback the correct dimensions